frontend/src/bundles/common/components/logo/logo.tsx

```diff
@@ -36,6 +36,29 @@ const Logo = ({
                     <img src={logoLabelSvg} alt="Label-BSATalents" />
                 </Grid>
             )}
+        </>
+    );
+};
+
+const Logo: React.FC<Properties> = ({
+    className = '',
+    isCollapsed = false,
+    link = { AppRoute },
+}) => {
+    return (
+        <Grid container>
+            {link === AppRoute.ROOT ? (
+                <Link
+                    to={link}
+                    className={getValidClassNames(styles.logo, className)}
+                >
+                    <BaseLogo isCollapsed={isCollapsed} />
+                </Link>
+            ) : (
+                <Grid className={getValidClassNames(styles.logo, className)}>
```

Comment:
```suggestion
                <Grid container item className={getValidClassNames(styles.logo, className)}>
```
styles.logo will not be applied if grid is not container

Reply:
True. I will fix this.